docs-site/docs/user/organization/members/membersship.md

```diff
@@ -0,0 +1,136 @@
+---
+sidebar_position: 1
+id: membersship
```

Comment:
Changing this to `members` caused the url to look like `http://localhost:4000/irida-next/docs/user/organization/members/` I'm assuming because the folder is also caused members. Changed the file name and id to `group-and-project-members` instead so we get a proper url `http://localhost:4000/irida-next/docs/user/organization/members/group-and-project-members`

Updated in a63a4d4ad7b141bcf3f735f6b628037c06a95b84